packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -274,6 +275,13 @@ export class FocusZone extends React.Component<IFocusZoneProps, {}> implements I
     return false;
   }
 
+  /**
+   * Gets the current active element of the focus zone.
+   */
+  public getActiveElement(): HTMLElement | null {
```

Comment:
@JasonGore do you think this is ok ? I added a new public API just for the unit tests.

Reply:
Unfortunately it adds to bundle size. We try to avoid adding API and functions used specifically for unit tests. Is there any way to test it without adding an API function?

Reply:
So, I had a test before that checked the `tabindex` attribute but I thought it was too brittle. But I guess that is better than increasing the bundle size for the test ?

Reply:
My 2c is that I can see this being generally useful. I haven't run across a scenario where it's needed, but this is a pretty low cost api.

Reply:
I'm not sure how we feel overall about setting it as a precedent but since private in javascript isn't private we could access it directly in the test rather than adding method as an alternative.

Reply:
Not sure how to go forward here. Let me know if you are strongly leaning towards removing the public function - I guess we could access the private function by casting to `any` ?

Reply:
@JasonGore do you have an opinion/preferred way of doing things?

Reply:
David says it could be useful, but I guess I would just lean towards adding it once it's needed, not just for a unit test... unless David knows of cases it would enable now.

Reply:
+1 I'm not a huge fan of adding a function for a unit test...

---
In reply to: [277366008](https://github.com/OfficeDev/office-ui-fabric-react/pull/8777#discussion_r277366008) [](ancestors = 277366008)

Reply:
Ok, removing the public API call. Instead, using an explicit cast to test the value of the private field. I am not sure if that is fine. Let me know.